app/components/NavigationCard.tsx

```diff
@@ -0,0 +1,43 @@
+import Link from "next/link";
+import Card from "./Card";
+import cn from "@/lib/helpers/cn";
+
+/**
+ * A Card containing a main navigation route in the main page.
+ * 
+ * @param title The Card title.
+ * @param body The Card's body text.
+ * @param href Where the Card navigates to.
+ * @param adornment A Small box indicating an ongoing event. Append like a property. 
+ */
+export default function NavigationCard({
+  adornment,
+  title,
+  body,
+  href,
+}: {
+  adornment?: React.ReactNode;
```

Comment:
On second thought, I think the `adornment` might work better as `children`. I know the specs said otherwise but…what do you think?